tests/DateTimeFactoryTest.php

```diff
@@ -81,6 +81,21 @@ public function it_creates_a_date_time_object_from_a_valid_atom_string(string $g
         $this->assertEquals($expected, $datetime);
     }
 
+    /**
+     * @test
+     */
+    public function it_creates_a_date_time_object_from_a_cdb_format(): void
+    {
+        $dateTime = DateTimeFactory::fromCdbFormat('2014-06-30T11:48:26');
+        $this->assertEquals(
+            new DateTimeImmutable(
+                '2014-06-30T11:48:26',
```

Comment:
Maybe we also need to check if it works without the `T` because I think that is why `?` is used.

Reply:
It's kinda hard, to still find all the formats that were used in `CdbXml` 🫣 I've only found ones with a `T` but all add some more testData.